### DISCUSSION TITLE
Error in document type ‘plain’ for 'nova/fct’

### DISCUSSION THREAD

**Discussion by Bailonis** [joaomlourenco/novathesis#419] [Q&A] [answered]
I put this in Config/1_novathesis.tex

docdegree=plain 

And got the respective error:

![image](https://user-images.githubusercontent.com/72702458/223765989-bd1abe2d-f26e-4b11-8370-cbefacf2f2dc.png)

**Comment by joaomlourenco** [comment, marked as answer]
@Bailonis I just uploaded a new version that fixes the problems with the "plain" document type (it will use the "red" color for the cover, but can be changed).

You may either download the full template again, or just the two changed files:
-  `NOVAthesisFiles/Schools/nova/fct/nova-fct-defaults.ldf`
- `novathesis.cls`

Please let me know if it worked for you.
